daemons/attrd/attrd_messages.c

```diff
@@ -180,6 +206,14 @@ handle_update_request(pcmk__request_t *request)
         return NULL;
 
     } else {
+        /* See comment on identical block in handle_clear_failure_request */
+        if (request->xml->children != NULL && ATTRD_SUPPORTS_MULTI_MESSAGE(minimum_protocol_version) &&
+            attrd_request_has_sync_point(request->xml)) {
+            crm_warn("Ignoring sync point in request from %s because not all nodes support it",
+                     pcmk__request_origin(request));
+            remove_sync_point_attribute(request->xml, NULL);
+        }
```

Comment:
might as well functionize it (or make it part of attrd_request_has_sync_point())

Reply:
Functionized.  I still need to test this out for real before I'm willing to remove the draft status.